.golangci.yml

```diff
@@ -57,6 +57,9 @@ linters-settings:
             recommandations:
               - io
               - os
+  revive:
+    rules:
+      - name: use-any
```

Comment:
Unfortunately, this is not the way to do with revive.

Or at least, it's not doing exactly what you want

Take a look here

https://github.com/ccoVeille/golangci-lint-config-examples/blob/4480ec8754d46e9993333847e6597141e00d408e/03-safe%2F.golangci.yml#L68-L139